stable/storageos-operator/app-readme.md

```diff
@@ -27,6 +29,8 @@ additional capacity, features and support plans contact sales@storageos.com.
 * High Availability - synchronous replication insulates you from node failure.
 * Delta Sync - replicas out of sync due to transient failures only transfer
     changed blocks.
+* Multiple AccessModes - dynamically provision ReadWriteOnce or ReadWriteMany
+  volumes
```

Comment:
Please add a period at the end of the line.

Reply:
Please add one more level of indentation to maintain consistency with other points.